### DISCUSSION TITLE
Copy/paste on Linux - Ctrl-V pastes, but select and Ctrl-C doesn't copy.

### DISCUSSION THREAD

**Discussion by John-Nagle** [emilk/egui#4462] [Q&A]
Copy/paste on Linux - Ctrl-V pastes the system clipboard into a text edit field, but select and Ctrl-C doesn't copy. Is there something I need to enable?

Issues indicate lots of bugs reported and fixed in this area, but current status seems to be that it's supposed to work.

Can reproduce with https://github.com/John-Nagle/ui-mock
which is my dummy game UI test - brings up the whole 3D stack and EGUI, but all it draws is a cube and some menus.

egui = "^0.26", on Rend3/WGPU/Vulkan/Linux. Default features.

Ubuntu 22.04 LTX, X-Windows.

**Comment by itsNiccoloSabatini** [comment]
are u using native or WASM32?

**Comment by John-Nagle** [comment]
egui = "^0.26", on Rend3/WGPU/Vulkan/Linux. Default features.

Ubuntu 22.04 LTS, X-Windows.

**Comment by SOF3** [comment]
Same on eframe on wasm32 on chrome 143. Ctrl-V pastes correctly but Ctrl-C does not do anything.

**Comment by itsNiccoloSabatini** [comment]
@SOF3 the wasm32 version need HTTPS to allow copy and paste, maybe LocalHost is ok too. Due to new browser security regulations Copying through JavaScript code is strictly forbidden, and the copy for egui wasm32 is triggered by CTRL+C but it's actually a JS code that put the text into the OS clipboard, that leaves us with the browser forbidding you to copy if the context isn't secure.
